src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -31,6 +31,9 @@
 
 #include "common/JsonHelper.h"
 
+namespace ngsiv2
```

Comment:
A first step in the issue https://github.com/telefonicaid/fiware-orion/issues/1290 solution...